ui/src/views/settings/SettingsMailboxes.vue

```diff
@@ -98,6 +98,58 @@
             </cv-row>
           </cv-grid>
         </cv-tile>
+        <cv-tile light>
+          <cv-grid fullWidth class="no-padding">
+            <cv-row>
+              <cv-column>
+                <h4 class="mg-bottom-md">
+                  {{ $t("settings_mailboxes.sharedseen") }}
+                  <cv-tooltip
```

Comment:
Let's use `cv-interactive-tooltip`: it's more mobile friendly since it doesn't require hover